addons/hr_maintenance/wizard/hr_departure_wizard_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="hr_departure_wizard_view_form" model="ir.ui.view">
+        <field name="name">hr.departure.wizard.view.form.extend</field>
+        <field name="model">hr.departure.wizard</field>
+        <field name="inherit_id" ref="hr.hr_departure_wizard_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='activities_label']" position="attributes">
+                <attribute name="invisible">0</attribute>
+            </xpath>
+            <xpath expr="//div[@id='activities']" position="attributes">
+                <attribute name="invisible">0</attribute>
+            </xpath>
```

Comment:
We need to add this xpath because in the main wizard, we don't have any activities. Therefore, to add termination activities, we need to make the label for activities and activities visible. This way, we can add activities, and they will be visible in the wizard. Additionally, even in the main wizard, there are comments indicating that we need to set the invisible  to 0 during inheritance.